ksqldb-engine/src/test/java/io/confluent/ksql/planner/LogicalPlannerTest.java

```diff
@@ -193,6 +192,7 @@ public void testSimpleRightJoinFilterLogicalPlan() {
     assertThat(rightSource.getSources().get(0), instanceOf(PreJoinRepartitionNode.class));
   }
 
+  @Ignore
```

Comment:
Looks like the latest commit adds this Ignore annotation to a number of unit tests. I'm guessing you have plans to address these?